com.unity.netcode.gameobjects/Runtime/Messaging/Messages/RpcMessages.cs

```diff
@@ -66,7 +66,10 @@ public static unsafe bool Deserialize(ref FastBufferReader reader, ref NetworkCo
         public static void Handle(ref NetworkContext context, ref RpcMetadata metadata, ref FastBufferReader payload, ref __RpcParams rpcParams)
         {
             var networkManager = (NetworkManager)context.SystemOwner;
-            var networkObject = networkManager.SpawnManager.SpawnedObjects[metadata.NetworkObjectId];
+            if (!networkManager.SpawnManager.SpawnedObjects.TryGetValue(metadata.NetworkObjectId, out var networkObject))
+            {
+                throw new Exception($"RPC called on a {nameof(NetworkObject)} that is not in the spawned objects list - please make sure the {nameof(NetworkObject)} is spawned before calling RPCs.");
```

Comment:
I like this.  There's part of me from my Java days that tells me we should use more specific exception types but...